business/jaeger.go

```diff
@@ -33,15 +33,23 @@ func (in *JaegerService) GetJaegerServices() (services []string, code int, err e
 	return client.GetJaegerServices()
 }
 
-func (in *JaegerService) GetJaegerTraces(ns string, srv string, query string) (traces []*jaegerModels.Trace, code int, err error) {
+func (in *JaegerService) GetJaegerSpans(namespace, service, startMicros, endMicros string) ([]jaeger.Span, error) {
```

Comment:
Minor, but I see some APIs using the "jaeger." package and other the imported "jaegerModels."
It would be nice to some kind of alignment even if that means just to create an alias to some "jaegerModels" to consolidate models used for the business logic.

Reply:
This `jaeger` is a package defined in Kiali whereas `jaegerModels` is an import alias of a package from jaeger lib, so, how would we do? Alias everything we use from jaeger lib within our jaeger package?